src/services/AudiusBackend.js

```diff
@@ -2438,6 +2440,20 @@ class AudiusBackend {
     }
   }
 
+  static async getRandomFeePayer() {
+    try {
+      return fetch(`${IDENTITY_SERVICE}/solana/random_fee_payer`, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json'
+        }
+      }).then(res => res.json())
+    } catch (err) {
+      console.error(err.message)
+      return { error: true }
+    }
+  }
+
```

Comment:
Let's have this live in libs - should be a method in the services/identity, and then probably called via SolanaWeb3Manager